src/storage/btree.c

```diff
@@ -22620,14 +22631,25 @@ btree_advance_and_find_key (THREAD_ENTRY * thread_p, BTID_INT * btid_int, DB_VAL
 	      assert (node_header->node_level == 1);
 #endif
 	      *advance_to_page = bcb_area;
+	      perfmon_inc_stat (thread_p, PSTAT_BT_NUM_COPY_LEAF_NOLATCH_SUCCESS);
+	      pgbuf_copy_log ("pgbuf_copy_page: Successfully copied B-tree page (%d, %d) \n",
+			      child_vpid.volid, child_vpid.pageid);
 	    }
 	  else
 	    {
+	      perfmon_inc_stat (thread_p, PSTAT_BT_NUM_COPY_LEAF_NOLATCH_FAILED);
```

Comment:
I'd like to separate "retry" and "failure". 

Please add a metric to count the number of retries.